docs/modules/ROOT/assets/attachments/slides_embedded.html

```diff
@@ -233,7 +233,7 @@
 <div class="paragraph"><p><a href="mailto:adrian@vshn.ch">adrian@vshn.ch</a></p></div>
 <link rel="stylesheet" href="lib/asciinema/asciinema-player.css">
 <script src="lib/asciinema/asciinema-player.js"></script></section>
-</div></div><script src="https://cdnjs.cloudflare.com/ajax/libs/reveal.js/3.7.0/lib/js/head.min.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/reveal.js/3.7.0/js/reveal.js"></script><script>Array.prototype.slice.call(document.querySelectorAll('.slides section')).forEach(function(slide) {
+</div></div><script src="https://cdnjs.cloudflare.com/ajax/libs/reveal.js/3.7.0/lib/js/head.min.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/reveal.js/4.0.2/js/reveal.js"></script><script>Array.prototype.slice.call(document.querySelectorAll('.slides section')).forEach(function(slide) {
```

Comment:
Doesn't this also need to be bumped?
```suggestion
</div></div><script src="https://cdnjs.cloudflare.com/ajax/libs/reveal.js/4.0.2/lib/js/head.min.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/reveal.js/4.0.2/js/reveal.js"></script><script>Array.prototype.slice.call(document.querySelectorAll('.slides section')).forEach(function(slide) {
```

Reply:
Argh, no, none of this must happen. The bump from 3 to 4 breaks existing HTML tags & CSS classes. better close this PR directly… thanks for the tip, but no thanks :)